blockchain/example/test_events.go

```diff
@@ -0,0 +1,31 @@
+package main
+
+import (
+	"log"
+
+	"github.com/sonm-io/core/blockchain"
+)
+
+func main() {
```

Comment:
It is not a test (like unit test), so remove a `test` word from a file name.
Just an `example/events.go` is OK, the name shows that this example file contains something about events.